Plugins/Flow.Launcher.Plugin.Explorer/Search/SearchManager.cs

```diff
@@ -34,17 +34,20 @@ internal List<Result> Search(Query query)
 
             var querySearch = query.Search;
 
-            var quickFolderLinks = quickFolderAccess.FolderList(query, settings.QuickFolderAccessLinks, context);
-
-            if (quickFolderLinks.Count > 0 && query.ActionKeyword == settings.SearchActionKeyword)
-                return quickFolderLinks;
-
-            if (string.IsNullOrEmpty(querySearch))
-                return results;
-
             if (IsFileContentSearch(query.ActionKeyword))
                 return WindowsIndexFileContentSearch(query, querySearch);
 
+            var quickFolderLinks = quickFolderAccess.FolderListMatched(query, settings.QuickFolderAccessLinks, context);
+
+            if (quickFolderLinks.Count > 0)
+                results.AddRange(quickFolderLinks);
+            
+            // This allows the user to type the assigned action keyword and only see the list of quick folder links
+            if (settings.QuickFolderAccessLinks.Count > 0
+                && query.ActionKeyword == settings.SearchActionKeyword
+                && string.IsNullOrEmpty(query.Search))
+                    return quickFolderAccess.FolderListAll(query, settings.QuickFolderAccessLinks, context);
+
```

Comment:
the second if statement can be moved up before the definition of `quickFolderLinks`